plugins/instanceof-checkcast/src/main/java/org/qbicc/plugin/instanceofcheckcast/SupersDisplayTables.java

```diff
@@ -338,6 +338,11 @@ public int getNumberOfBytesInInterfaceBitsArray() {
     public int getInterfaceByteIndex(LoadedTypeDefinition cls) {
         Assert.assertTrue(cls.isInterface());
         IdAndRange idRange = typeids.get(cls);
+        try {
+            int rc = idRange.implementedInterfaceByteIndex();
+        } catch (NullPointerException e) {
+            System.out.println("NPE caught for cls: " + cls.getInternalName());
```

Comment:
Debugging code that should be removed?  Or if we are going to catch a NPE then flag as an error

Reply:
aah! thanks for pointing this out. I will update the commit to remove it.